modules/t/test-genome-DBs/test_master/compara/genome_db.txt

```diff
@@ -1,5 +1,8 @@
+1	9606	homo_sapiens	GRCh36	2009-07-Ensembl	0	0	\N	\N	Human	\N	79	\N
```

Comment:
This test db is used by another test, which tests whether it can add/update/release genome dbs, including the ones you've added here. These changes may disrupt the other test (`MasterDatabase.t`).

Reply:
I did check that the other tests using this database passed with this change, and Travis doesn't seem to complain either :smile: